rc/v.kak

```diff
@@ -0,0 +1,74 @@
+##
+
+## v.kak by Conscat
+##
+
+# https://vlang.io
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .+\.(v|vv|vsh) %{
+    set-option buffer filetype v
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=v %{
+    require-module v
+    set-option window static_words %opt{v_static_words}
+    add-highlighter window/v ref v
+}
+
+hook global WinSetOption filetype=(?!v).* %{
+    remove-highlighter window/v
+}
+
+declare-option str comment_line
+hook global BufSetOption filetype=v %{
+    set-option buffer comment_line '//'
+    set-option buffer comment_block_begin '/*'
+    set-option buffer comment_block_end   '*/'
```

Comment:
Since `comment.kak` might not be loaded and provide with these two options, they will also have to be declared. Maybe you could declare them within the hook too, since this scope is only used to add support for commenting.